test/other/test_standalone_syscalls.cpp

```diff
@@ -15,20 +16,27 @@ int main() {
   assert(fd_out == 1);
   int fd_err = open("/dev/stderr", 0);
   assert(fd_err == 2);
+
   // ioctl and fcntl fail
   int i = ioctl(1, 0);
   assert(i < 0);
   int f = fcntl(1, 0);
   assert(f < 0);
+
   // write to standard streams works.
   write(1, "hello, world!", 5);
   write(1, "\n", 1);
+
   // emscripten_log API works
   emscripten_console_log("log");
   // warnings/errors go to stderr
   emscripten_console_warn("warn");
   emscripten_console_error("error");
   emscripten_console_log("log2");
+
   // check we can call this, but the test doesn't check the output
   emscripten_get_now();
+
+  raise(SIGUSR1);
```

Comment:
Hmm, doesn't this cause a runtime error of some kind? If not what does it do? (maybe add a comment)

Reply:
It does nothing because we haven't registered a handler.    It just tests that `raise` can be called in standlone more without triggering the `__call_handler` JS import.

Reply:
I see, thanks!